test/command_line/test_search_beam_position.py

```diff
@@ -31,7 +31,7 @@ def test_search_i04_weak_data_image_range(mocker, run_in_tmpdir, dials_regressio
         "image_range=1,10",
         "image_range=251,260",
         "image_range=531,540",
-        "n_macro_cycles=2",
+        "n_macro_cycles=4",
```

Comment:
Just for interest, why change in `n_macro_cycles`?

Reply:
It appears for this weak dataset that the new indexing code takes a few more cycles to converge.